SDL2_mixerConfig.cmake.in

```diff
@@ -85,6 +87,10 @@ include(CMakeFindDependencyMacro)
         find_dependency(vorbisfile)
     endif()
 
+    if(SDL2MIXER_WAVPACK AND NOT SDL2MIXER_VENDORED AND NOT TARGET WavPack::WavPack)
```

Comment:
Does CI need any changes after this?

Reply:
No, it's built by default. CMake fails when wavpack is not available.

Reply:
The only thing missing for wavpack is an `Android.mk` make file,
and I forgot to add wavpack support for creating an android archive with prefab support.

Reply:
Can you do it, or make a later thing?

Reply:
Added in https://github.com/libsdl-org/WavPack/commit/7c4391c076f68a8d4e07383dcf52aa0806da0388 and e426798

Reply:
Great. Anything else left about build systems, except for Xcode and MSVC project files?

Reply:
Great!